tmgmt_zanata.module

```diff
@@ -47,3 +47,42 @@ function tmgmt_zanata_poll_translations($form, &$form_state) {
   $job = $form_state['tmgmt_job'];
   $job->getTranslatorController()->getConnector($job)->pollTranslations();
 }
+
+/**
+ * Implements hook_cron().
+ *
+ * Cron based callback used to poll for translations
+ */
+function tmgmt_zanata_cron(){
+    if (variable_get('poll_translations_during_cron', TRUE)) {
+        $query = new EntityFieldQuery();
+        $result = $query->entityCondition('entity_type', 'tmgmt_job')->propertyCondition('state',
+            TMGMT_JOB_STATE_ACTIVE)->propertyCondition('translator', 'zanata')->execute();
+
+        if (!empty($result['tmgmt_job'])) {
+            $active_translation_jobs = array_keys($result['tmgmt_job']);
+            $queue = DrupalQueue::get('poll_trans');
+            foreach ($active_translation_jobs as $job_id) {
+                $queue->createItem($job_id);
+            }
+        }
+    }
+}
+
+/*
+ * Implements hook_cron_queue_info().
+ */
+function tmgmt_zanata_cron_queue_info(){
+    $queues['poll_trans'] = array(
+        'worker callback' => 'poll_translation',
+        'time' => 120,
```

Comment:
This may make a good configuration option as well. It could be a choice of a few options such as (2 minutes, 10 minutes, 1 hour, 8 hours, 1 day), or just enter a number of minutes. It should not accept a value of less than 2 minutes though.